Documentation/ExtensionArchitecture/CreateNewExtension/Index.rst

```diff
@@ -43,16 +43,16 @@ comes with its own BE module:
    The Domain Modeller screen of the Extension Builder. The comfort of building
    your model with drag and drop.
 
-Note that this tool is not a complete editor. It helps you creating the scaffolding
-of your extension, generating the necessary files. It's then up to you to fill these
-with the relevant code.
+Note that this tool is not a complete editor. It helps you to create the scaffolding
+of your extension and to generate all necessary PHP files. It is then up to you to complete these files
```

Comment:
While this is definitely an improvement, the part "complete these files with your intended code" reads a bit awkward. In any case, I would drop "intended" because it is not just intended code, it is actual code.

Maybe: "You must then complete the source code using your preferred editor or IDE.